node_modules/gh-core/lib/api.js

```diff
@@ -74,6 +74,9 @@ var init = module.exports.init = function(config, callback) {
                         return callback(err);
                     }
 
+                    // Initialise the Pattern listener
+                    require('gh-series/lib/internal/patterns/cambridge/listener');
```

Comment:
That's already what happens. This require just registers the listener.

The listener will only perform aggregation for timetable apps.